sources/packages/backend/apps/db-migrations/src/migrations/1711061241340-AddSFASPartTimeApplicationPYMaximumColumns.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddSFASPartTimeApplicationPYMaximumColumns1711061241340
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Add-col-py-maximum-support.sql",
```

Comment:
Minor: `Add-cols-py-maximum-support.sql` since we are adding multiple column.

Reply:
`cols` sounds good to me.

Reply:
I think I will go with `Add-cols-sfas-pt-awards`.